src/components/Header/index.jsx

```diff
@@ -83,14 +83,6 @@ const Header = ({
   hasSidebarToggle, enterpriseName, enterpriseLogo, enterpriseSlug,
 }) => {
   const user = getAuthenticatedUser();
-  useEffect(() => {
-    // if there is no email, the user data has not been hydrated
-    if (user && !user.email) {
-      hydrateAuthenticatedUser();
-    }
-    // rehydrate if the username changes
-  }, [user?.username, user?.id]);
-
```

Comment:
This results in a duplicate call to hydrate the user's metadata that is legacy since the `@edx/frontend-platform` refactor.